### PR TITLE
Release v0.13.0

### PR DESCRIPTION
**Pull request by wischli**
# Description

* Removes migrations which are expected to have been executed
* Bumps spec version to 1300
* Updates deps to enable metadata hash check as recommended [on Element](https://matrix.to/#/!pYjTIdxhocIQxcOXdb:matrix.parity.io/$rgj5rvCJXXitBp49-8Umx6bD-VMqoanZFefY7vgiSaw?via=matrix.parity.io&via=matrix.org&via=parity.io)
* Updates srtool to latest version
* Adds `on-chain-release-build` to srtool build opts
  * Longterm we should switch to using the explicit github action: https://github.com/polkadot-fellows/runtimes/blob/bb52c327360d1098d3b3d36f4eafb40a74636e80/.github/workflows/release.yml#L59

*Pipeline enhancements*
- Limit sanity_checks to 90m to avoid consuming minutes on stuck jobs
- Modify wasm build to output srtool information
- Upload artifacts when creating a release (wasm and json summary)
- Upload json summary to GCP bucket too
- Add a manual workflow to delete extra containers from GH registry (verified working)
- Remove updating the release with the docker tag as appended comment (preferred to do manually)

# Checklist:

- [x] I have added Rust doc comments to structs, enums, traits and functions
- [x] I have made corresponding changes to the documentation
- [x] I have performed a self-review of my code
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: runtime/development/src/migrations.rs
Comment (wischli) [inline runtime/development/src/migrations.rs:27]: NOTE: Still needed because we haven't upgraded Dev and Demo to 1200.

File: Cargo.toml
Comment (lemunozm) [inline Cargo.toml:56]: Why a major version instead of 0.12.1?

**Review by lemunozm — APPROVED** [review]
LGTM

File: Cargo.toml
Comment (wischli) [reply Cargo.toml:56]: That's something I battled with internally myself. I thought this would be cleaner because v0.12.0 was related to enabling OpenGov on Altair.

Since this enables the metadata hash check, I think it deserves its major version. For similar releases in the future, I will take the minor version approach then since you seem to prefer that as well.

**Review by mustermeiszer — DISMISSED** [review]
That was fast! LGTM!

File: runtime/altair/Cargo.toml
Comment (lemunozm) [inline runtime/altair/Cargo.toml:160]: I think it should be `"substrate-wasm-builder/std"` to enable also std, if not, only the basic crate is added

File: runtime/altair/Cargo.toml
Comment (lemunozm) [inline runtime/altair/Cargo.toml:463]: Why the `?`?. is not your intention to always set the `metadata-hash` of `substrate-wasm-builder` each time you set `metadata-hash`

**Comment by wischli** [comment]
@lemunozm Seems like WASM building succeeds now. Can you please to the nth re-approval so we can get this merged finally? Thanks for the patience! And thanks @gpmayorga for putting so much energy in improving our WASM CI.

File: .github/workflows/build-docker.yml
Comment (lemunozm) [inline .github/workflows/build-docker.yml:122]: Tech-debt, should be maintained in the repo or removed? @gpmayorga

**Review by lemunozm — APPROVED** [review]
Let's merge this! Thanks for your work on this 🚀

**Comment by wischli** [comment]
@gpmayorga We need your approval as CI code owner

File: .github/workflows/build-docker.yml
Comment (wischli) [reply .github/workflows/build-docker.yml:122]: Good catch. IMO, we can remove this!

**Review by gpmayorga — APPROVED** [review]

**Pull request merged**
